### DISCUSSION TITLE
How can I use Firebase + Facebook?

### DISCUSSION THREAD

**Discussion by BrodySchuman** [quasarframework/quasar#9196] [Quasar App Extensions]
Hello everyone. I'm new to Quasar Framework and Vue.js. I want to make a login with facebook using Firebase. I followed this tutorial: https://dev.to/quasar/to-the-stars-with-quasar-firebase-user-profile-319l

How can I add facebook authentication to my app after following this tutorial? I can't find any tutorials similar to this, and I tried searching and couldn't find anything..

Could anyone help point me in the right direction please?

**Comment by ontwikkelfabriek** [comment]
I assume you have the firebase app from the article fully up and running.

See here the official firebase docs:
https://firebase.google.com/docs/auth/web/facebook-login

Here a nice article about using firebase-ui in Vue:
https://softauthor.com/firebaseui-vue-facebook-google-auth

FirebaseUI ( for web):
https://firebase.google.com/docs/auth/web/firebaseui
https://github.com/firebase/firebaseui-web

Here's an article about setting up vue with firebase and facebook without using FirebaseUI:
https://blog.logrocket.com/vue-firebase-authentication/
